lib/beaker-hostgenerator/data.rb

```diff
@@ -185,6 +185,17 @@ def osinfo
                           'box' => 'debian/bullseye64',
                         },
                       },
+                      'debian11-AARCH64' => {
+                        general: {
+                          'platform' => 'debian-11-aarch64',
+                        },
+                        abs: {
+                          'template' => 'debian-11-arm64',
+                        },
```

Comment:
This should be redundant since it's generated by:
https://github.com/voxpupuli/beaker-hostgenerator/blob/f47bca25f12c53161c923cdda7c8f8462b9f4594/lib/beaker-hostgenerator/hypervisor/abs.rb#L30-L45

Reply:
Ah yes, I was a bit confused because the `template` key was missing from the generated fixtures, but specifying the `abs` hypervisor does indeed generate that key as expected.